config/routes.rb

```diff
@@ -427,8 +427,12 @@
 
     get "/page/:slug" => "pages#show"
 
+    # TODO: [forem/teamsmash] removed the /p/information view and added a redirect for SEO purposes.
+    # We need to remove this route in 2 months (11 January 2021).
```

Comment:
Why 2 months?

Reply:
I think this is pretty arbitrary. Enough time to let stuff stop linking to it, not so much time to be a big deal if eventually there are broken links in a few months.

Reply:
sorry @mstruve just seen this now. What Ben said :)